### PR TITLE
Update Leadership Page 2019-2020

### PR DESCRIPTION
**Pull request by jambamslamlord**

### CODE REVIEW

File: _data/leads.yml [hunk 1]
```diff
@@ -1,39 +1,50 @@
...
-- name: William Hou
-  subteam: Strategy
-  image: william.jpg
-  bio: William is a junior and co-strategy lead along with Sander on the South Eugene Robotics Team. This is William’s third year on the team, and he is also a part of the software subteam. He loves watching robots at competitions and helping organize scouting so that the team has useful data before each match. Outside of school and robotics, William enjoys biking, talking with his friends, and playing online games.
+- name: "Ben Weber"
+  subteam: "Co-Safety"
+  image: Ben.jpg
+  bio: "Ben Weber is a senior and a co-lead of the safety team. Ben has been involved with FIRST since FIRST Lego League. On SERT, Ben only cares about one thing, and one thing only: the safety for everyone around. Outside of robotics, Ben is on that grind playing water polo and French Horn. In his free time, Ben is busy reading, watching Netflix, and chilling with friends."
+  
+- name: "Mustafa aljabery"
```
Comment (will-hou) [inline _data/leads.yml]:
```suggestion
- name: "Mustafa Aljabery"
```

File: _pages/leadership.md [hunk 2]
```diff
@@ -84,3 +84,6 @@ and successful matches. This includes creating and guiding the team through a
 process to design the best robot our team can build for this year's game,
 testing the prototype and the robot in thorough and exciting ways, working
 on match-strategy, and organizing scouting.
+
+### Safety
+The Safety sub team is dedicated to making FIRST and SERT a safer and better place for all. The Safety Subteam is responsible for enforcing safe practices, educating team members and the public about safety, and managing a safe environment for all. At SERT, Safety always comes FIRST!
```
Comment (will-hou) [inline _pages/leadership.md]:
```suggestion
The safety subteam is dedicated to making FIRST and SERT a safer and better place for all. It is responsible for enforcing safe practices, educating team members and the public about safety, and managing a safe environment for all. At SERT, safety always comes FIRST!
```

Comment (will-hou) [reply _pages/leadership.md]: Just for consistency with the rest of the subteam descriptions

File: _data/leads.yml (hunk 1 above)
Comment (andrewda) [reply _data/leads.yml]: Good catch

**Pull request merged**